stable/prometheus-operator/templates/grafana/dashboards/k8s-coredns.yaml

```diff
@@ -1,5 +1,5 @@
 # Added manually, can be changed in-place.
-{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled .Values.coreDns.enabled }}
+{{- if and (semverCompare ">=1.11.0" .Capabilities.KubeVersion.GitVersion) (semverCompare "<1.14.0" .Capabilities.KubeVersion.GitVersion) .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled .Values.coreDns.enabled }}
```

Comment:
I re-ran the script and it appears to have removed this line, so now helm is trying to provision this dashboard twice

Reply:
Strange i had to add the semver check manually since this dashboard was `# Added manually, can be changed in-place.`  to fix the first test run failure